src/styles/_header.scss

```diff
@@ -171,6 +179,9 @@
           );
           &:hover {
             border: none;
+            &::after {
+              border: 3px solid color('secondary');
+            }
```

Comment:
Which lines make the page not jump on a button hover?

Reply:
Setting `$border-color` and `$border-color-hover` in the `button-theme` mixins in the global styles file